node/nanoexpress/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM node:12.6.0
+
+RUN npm -g install pm2
+
+WORKDIR /usr/src/app
+
+COPY app.js package.json ./
+
+RUN apk update && apk add --no-cache git
```

Comment:
no `apk` `node` image is `debian` base not `alpine`

Reply:
I changed Dockerfile image for faster download and lighter Node.js image, so it is works

Reply:
I prefer keeping the official image :stuck_out_tongue: not because using `slim` is not an advantage `here`, but because of making uniform containers
* if `node` community prefer **debian** based, its their choice, not ours
* not all languages has `slim` variants (making unfair comparison between languages)
* I'll update this project to `kubernetes` (and it is better to use `official` images)

Reply:
Okay, thanks. I'll updated Dockerfile, but cannot test due of traffic (sorry for this)